lib/src/models/multireddit.dart

```diff
@@ -3,13 +3,428 @@
+    };
+    return await reddit.post(url, data);
+  }
+
+  /// Delete this [Multireddit].
+  ///
+  /// Does not refresh current instance of Multireddit, as it has been deleted.
+  Future delete() async {
+    await reddit.delete(apiPath['multireddit_base'] + _infoPath);
+  }
+
+/*
+  /// Remove a [Subreddit] from this [Multireddit].
+  ///
+  /// [subreddit] contains the name of the subreddit to be deleted.
+  Future remove({String subreddit, Subreddit subredditInstance}) async {
+    subreddit = subredditInstance?.displayName;
+    if (subreddit == null) return;
+    final url = apiPath[_kMultiredditUpdate]
+        .replaceAll(_multiredditRegExp, _name)
+        .replaceAll(User.userRegExp, _author)
+        .replaceAll(_subredditRegExp, subreddit);
+    final data = {'model': "{'name': $subreddit}"};
+    await reddit.delete(url, body: data);
+  }
+
+  /// Rename this [Multireddit].
+  ///
+  /// [newName] is the new display for this [Multireddit].
+  /// The [name] will be auto generated from the displayName.
+  Future rename(newName) async {
+    final url = apiPath["multireddit_rename"];
+    final data = {
+      _kFrom: _infoPath,
+      _kTo: newName,
+    };
+    final response = await reddit.post(url, data);
+    _name = newName;
+    return response;
+  }
+
+  /// Update this [Multireddit].
+  Future update(
+      {final String displayName,
+      final List<String> subreddits,
+      final String descriptionMd,
+      final IconName iconName,
+      final Color color,
+      final Visibility visibility,
+      final WeightingScheme weightingScheme}) async {
+    final newSettings = {};
+    if (displayName != null) {
+      newSettings[_kDisplayName] = displayName;
+    }
+    /*
+      The Reddit api requires we use the following JSON for a subreddit:
+      {'name': theNameOfTheSubreddit}, for each subreddit in the list.
+      For this reason we do a map here to convert back to api format.
+     */
+    final newSubredditList =
+        subreddits?.map((item) => {'name': item})?.toList();
+    if (newSubredditList != null) {
+      newSettings[_kSubreddits] = newSubredditList;
+    }
+    if (descriptionMd != null) {
+      newSettings["description_md"] = descriptionMd;
+    }
+    if (iconName != null) {
+      newSettings["icon_name"] = iconNameToString(iconName);
+    }
+    if (visibility != null) {
+      newSettings[_kVisibility] = visibilityToString(visibility);
+    }
+    if (weightingScheme != null) {
+      newSettings[_kWeightingScheme] =
+          weightingSchemeToString(weightingScheme);
+    }
+    if (color != null) {
+      newSettings["key_color"] = color.toHexColor().toString();
+    }
+    //Link to api docs: https://www.reddit.com/dev/api/#PUT_api_multi_{multipath}
+    final res = await reddit.put(_infoPath, body: newSettings.toString());
+    final Multireddit newMulti = new Multireddit.parse(reddit, res['data']);
+    _name = newMulti.displayName;
+    _subreddits = newMulti._subreddits;
   }
+  */
```

Comment:
You've got nested block comments here so I don't think you commented out everything you meant to.